### PR TITLE
Dev valbano 727

### PR DESCRIPTION
**Pull request by VitoAlbano**
#741 - user info component

### CODE REVIEW

File: ng2-components/ng2-alfresco-upload/demo/index.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (magemello) [inline ng2-components/ng2-alfresco-upload/demo/index.html]: can you remove it?

File: ng2-components/ng2-alfresco-userinfo/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+language: node_js
```
Comment (magemello) [inline ng2-components/ng2-alfresco-userinfo/.travis.yml]: this file is not needed, you can delete it

File: ng2-components/ng2-alfresco-userinfo/demo/wsrv-config.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (magemello) [inline ng2-components/ng2-alfresco-userinfo/demo/wsrv-config.json]: do we need this file?

File: ng2-components/ng2-alfresco-userinfo/demo/index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Angular 2 TaskList - Demo</title>
```
Comment (magemello) [inline ng2-components/ng2-alfresco-userinfo/demo/index.html]: the title is wrong

File: ng2-components/ng2-alfresco-userinfo/demo/package.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
+{
+  "name": "ng2-activiti-tasklist-demo",
+  "description": "Alfresco Angular2 Task List Component - Demo",
```
Comment (magemello) [inline ng2-components/ng2-alfresco-userinfo/demo/package.json]: name and description are wrong

File: ng2-components/ng2-alfresco-userinfo/demo/wsrv-config.json (hunk 3 above)
Comment (VitoAlbano) [inline ng2-components/ng2-alfresco-userinfo/demo/wsrv-config.json]: removed

File: ng2-components/ng2-alfresco-userinfo/demo/index.html (hunk 4 above)
Comment (VitoAlbano) [inline ng2-components/ng2-alfresco-userinfo/demo/index.html]: fixed

File: ng2-components/ng2-alfresco-userinfo/demo/package.json (hunk 5 above)
Comment (VitoAlbano) [inline ng2-components/ng2-alfresco-userinfo/demo/package.json]: Fixed

**Comment by magemello** [comment]
You need to add also your component to the .travis.yml file

**Comment by VitoAlbano** [comment]
Done.

**Pull request merged**